test/fg-expect/view.chain-request.json

```diff
@@ -0,0 +1,8 @@
+{
+    "webserver" : {
+        "views/chain.request": {
+            "view": "test.chain.request"
```

Comment:
You can embed this in the .fg script as well. May be simpler

Reply:
But actually, why not just use `test.chain.request` directly in the .fg test script?

Reply:
Just forget we can do that